### DISCUSSION TITLE
央央體育室

### DISCUSSION THREAD

**Discussion by thz0000** [1101-GS4719-C/Python-Community-GS4719-C#1740] [General]
近期學校頒布了一項規定：禁止在田徑場旁的空地打球，除了體育課和校隊練習。

這當然遭到許多同學撻伐，即使是以安全為考量，但若真的是為了安全，那不應該是全面禁止嗎？

為什麼這會引起如此大的反彈呢？因為根本原因是我們沒有足夠多的運動場地，但學校在沒有解決根本問題的情況下，直接剝奪了練球的空間。

而且，學校的室內場每一種都要錢，重點是還不便宜。真的很想問一句：學生是斂財工具嗎？

順便想請問一下大家，有什麼能增加運動場地的方法或打球的好去處嗎？因為感覺增建或調降室內場費用都不太可行...QQ

**Comment by Chengbosco** [comment]
NCU  NEVER CARE YOU
我也覺得讓校隊開後門很好笑，要這樣做其實就是代表根本沒有足夠的運動場地阿。
還好我是宅宅，不想運動。

**Comment by THH1118** [comment]
之前有一次買了排球室內場的票，結果裡面超級多人所以就一堆人只能用兩個場地報隊打，室內也超級悶，除了地板有好那麼一點基本沒什麼優點，真的沒有想花錢進去第二次。

**Comment by yue0903ZZ** [comment]
學校禁打球確實讓人無奈，主要是場地不夠又收費高，讓學生難以有運動空間。增加場地可考慮夜間開放校外運動場，或推動社區合作共用設施。
